content/release-10-16-0/applications-10-16-0-bundle/10-16-0-0.md

```diff
@@ -187,6 +187,17 @@ MTM-47676</td>
 <td>MTM-47517</td>
 </tr>
 
+<tr>
+<td>Cockpit</td>
+<td>The "Welcome" widget in the Cockpit application is now reliably displaying the <b>Register device</b> link as long as the "Devicemanagement" application is subscribed to the tenant.</td>

```

Comment:
is "Devicemanagement" correct? Not missing a space?

Reply:
No, this is the name of the application